source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs

```diff
@@ -36,7 +36,7 @@ public async Task<string> GetContainerImageForCluster()
 
             var tag = tagVersion?.ToString(2) ?? "latest";
 
-            return $"octopuslabs/k8s-workertools:{tag}";
+            return $"docker.packages.octopushq.com/octopusdeploy/kubernetes-agent-tools-base:{tag}";
```

Comment:
Are we pushing this image to dockerhub yet? Or are we going to wait until we've tested it more?

Reply:
I was thinking we'd wait until we've tested it and then just before we release we can push it to dockerhub.

Reply:
yep, I think that's a fair call. Can you put a ticket in Shortcut so we don't forget to do this?